application/matching/templates/match-db-dep.yaml

```diff
@@ -26,19 +26,10 @@ spec:
           containerPort: 7687
         resources:
 {{ toYaml .Values.matchdb.resources | indent 10 }}
-      #   securityContext:
```

Comment:
normally you did not change all those files.
will you try a git pull origin main to make sure we are up to date?